staging/src/k8s.io/client-go/examples/informer/README.md

```diff
@@ -0,0 +1,86 @@
+# Informer Example
+
+Informers provide a high-level API for creating custom controllers for Kubernetes resources.
+
+This particular example demonstrates:
+
+* How to write an Informer against a core resource type.
+* How to handle add, update and delete events.
+
```

Comment:
Could you link to the thirdparty resource example? Something like "The [thirdparty-resources](../third-party-resources) example demonstrates how to use informers against a thirdparty resource"